SETUP.md

```diff
@@ -92,7 +108,7 @@ CREATE TABLE IF NOT EXISTS gis
 ----
 
 ## Setting up Internal Cache Store
-- In order to connect with the internal cache, we need to add cache verticle configs available in [config-example.json](configs/config-example.json).
+- In order to connect with the internal cache, we need to add cache verticle configs available in [config-example.json](example-configs/configs/config.json).
```

Comment:
can be rephrased to 

- To connect to the internal cache, ensure the cache vertical configurations are added.

Reply:
solved